backend/overstroomik_service/main.py

```diff
@@ -20,16 +20,19 @@ async def read_root():
     return {"Hello": "World"}
 
 
-@app.get("/items/{item_id}")
-async def read_item(item_id: int, q: Optional[str] = None):
-    return {"item_id": item_id, "q": q}
-
-
-@app.post("/location", response_model=Response)
-async def dataservice(location: Location):
-    # location.resolve()
-    # something with data
-    return Response(location=location)
+@app.get("/by_location", response_model=FloodInfo)
+def by_location(
+    search_field: Optional[str] = None,
+    latitude: Optional[float] = None,
+    longitude: Optional[float] = None,
+) -> FloodInfo:
+    return FloodInfo(
```

Comment:
Should it do something in between?

Reply:
I will add a comment that it should do something in between, but there's nothing there yet.